data/categories.txt

```diff
@@ -2085,6 +2085,46 @@ sk:3Trhovisko|samoobsluha
 sw:Soko
 fa:بازار
 
+amenity-money_transfer
```

Comment:
Should it be added to the `@bank` search category?
Maybe not in this PR, but I'm just throwing an idea in.
ATM `@bank` is for `amenity-bank` only. And we have a separate `@atm` category.
I think money_transfer and bureau_de_change fits into the `@bank` category (or maybe rename it to "Money"?)
E.g. people might want to find a bank because they want to exchange or make a transfer. ATMs should probably stay in a separate category as it is now.

Reply:
There are pros and cons to both approaches.
What about people who like to find specific places like Transferwise, or something else, that is not supported yet?